admin/client/src/components/ListGroup/ListGroup.scss

```diff
@@ -56,27 +61,23 @@
   opacity: .6;
 
   &:hover {
-		opacity: 1;
+    opacity: 1;
+
+    .list-group-item__status {
+      opacity: 1;
+    }
 	}
+
+  .list-group-item__status {
+    opacity: 0;
+  }
 }
 
 .list-group-item__status {
   display: inline-block;
   transition: opacity .2s ease-in-out;
 }
 
-.list-group-item.active .list-group-item__status {
-	color: #FFF;
-	opacity: 1;
-}
-
-.list-group-item--inactive .list-group-item__status {
```

Comment:
Ok I remember why I removed the css and not the html. This feature has just been taken out of Alpha1 so we want it to remain hidden hence we don't want the opacity: 1 at all. I think I'll add a comment to explain this for future reference.